receiver/statsdreceiver/protocol/statsd_parser_test.go

```diff
@@ -20,15 +20,23 @@ import (
 
 	metricspb "github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1"
 	"github.com/stretchr/testify/assert"
+	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
 func Test_StatsDParser_Parse(t *testing.T) {
+	prevTimeNowFunc := timeNowFunc
+	timeNowFunc = func() int64 {
+		return 0
+	}
+	defer func() {
+		timeNowFunc = prevTimeNowFunc
+	}()
```

Comment:
```
t.Cleanup(
		func() {
			timeNowFunc = prevTimeNowFunc
		},
)
```

What do you think of doing this instead?

Reply:
Fixed.